Dimens.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Dimens.swift
+//  ios-sdk
+//
+//  Created by Sarah Lensing on 2/24/17.
+//  Copyright © 2017 Mapzen. All rights reserved.
+//
+
+import UIKit
+
+open class Dimens {
```

Comment:
1. Public would make more sense...they probably wont subclass it, just access the constants
2. Updated
3. Renamed to Dimensions
4. Documented